arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,7 +410,7 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
-        "provisioningState": {
+        "state": {
```

Comment:
@amarzavery 
I made a wrong change. We want to keep provisioningState in swagger and add new "state" property to let it used by SDK. So there won't be breaking change and REST API is accurately described as it is
@amarzavery @ravbhatnagar 
I would update reason why we need addition state field instead of reuse provisioningState soon